README.md

```diff
@@ -40,55 +40,87 @@ In your service worker you just need to use `importScripts` to load Shed
-// change it like so:
-shed.router.default = shed.cacheFirst;
+```javascript
+var myHandler = function(request, values, options) {
+	// ...
+}
+```
 
-// If you want some resources to be cached during the install event, specify
-// them with the precache method
-shed.precache(['/index.html', '/site.css', '/images/logo.png']);
+- `request` - Request object that triggered the `fetch` event
+- `values` - Object whose keys are the placeholder names in the URL pattern, with the values being the corresponding part of the request URL. For example, with a URL pattern of `'/images/:size/:name.jpg'` and an actual URL of `'/images/large/unicorns.jpg'`, `values` would be `{size: 'large', name: 'unicorns'}`
+- `options` - the options object that was used when creating the route (see the API docs below)
 
-// You can manually add or remove things from the cache
-shed.cache('/data/2014/posts.json');
-shed.uncache('/data/2013/posts.json');
+The return value should be a Response, or a promise for a Response. If another value is returned, or if the returned promise is rejected, the Request will fail.
 
-// You can pass options to most methods
-// The `debug` option turns on verbose console logging
-shed.router.get('/example/route/', shed.cacheFirst, {debug: true});
+### Built-in handlers
 
-// The `cache` option tells shed to use an alternative cache. Provide the string name of the cache, not a Cache object.
-shed.cache('/images/foo.png', {cache: 'my other cache'});
+There are 5 built-in handlers to cover the most common network strategies.
 
-// These can also be set globally
-shed.options.debug = true;
-shed.options.cache = 'main cache';
+#### `shed.networkFirst`
+Try to handle the request by fetching from the network. If it succeeds, store the response in the cache. Otherwise, try to fulfill the request from the cache. This is the strategy to use for basic read-through caching. Also good for API requests where you always want the freshest data when it is available but would rather have stale data than no data.
 
-```
+#### `shed.cacheFirst`
+If the request matches a cache entry, respond with that. Otherwise try to fetch the resource from the network. If the network request succeeds, update the cache. Good for resources that don't change, or for which you have some other update mechanism.
+
+#### `shed.fastest`
+Request the resource from both the cache and the network in parallel. Respond with whichever returns first. Usually this will be the cached version, if there is one. On the one hand this strategy will always make a network request, even if the resource is cached. On the other hand, if/when the network request completes the cache is updated, so that future cache reads will be more up-to-date.
+
+#### `shed.cacheOnly`
+Resolve the request from the cache, or fail. Good for when you need to guarantee that no network request will be made - to save battery on mobile, for example.
+
+#### `shed.networkOnly`
```

Comment:
Mention that this is the value initially used for `shed.router.default`. (It is, right?)